sci-libs/pg2plplot/pg2plplot-5.13.0.ebuild

```diff
@@ -14,19 +16,20 @@ SLOT="0"
 KEYWORDS="~amd64 ~x86"
 IUSE="X png postscript static-libs"
```

Comment:
The `USE` flags are a bit weird.
They don't seem to be used in the configuration of the current package?
Does it automagically pick up that the dependencies are built with said flag or not?
Consider this - I don't enable `png` and `postscript` flags on `pg2plplot` but I do build `plplot` with `cairo`.
The building of `pg2pltplot` will pick up that `plplot` has `cairo` and will build the `png` and `postscript` parts, but if I later rebuild `plplot` without `cairo` I will get preserved libraries that will need to be rebuilt using `@preserved-rebuild`.

My proposal - remove `png`, `postscript` and `X` flags and make the dependency be `plplot[cairo,fortran,X]`.